### PR TITLE
Our first ADR, about using ADRs

### PR DESCRIPTION
**Pull request by tkadlec**

### CODE REVIEW

File: docs/adr/0001-use-adrs.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+1. A developer creates an ADR document outlining an approach for a particular question/problem, with an initial status of "proposed".
+2. The core team reviews the ADR, discussing any concerns/alternatives. We'll update the ADR accordingly to reflect the additional context.
+3. Once the core team has reached consensus, we either mark the ADR as "accepted" or "rejected".
+4. Should we revisit an ADR at some point and reach a different conclusion, a new ADR should be created with the new context and rationale for the change, referencing the old ADR. Once the new ADR is accepted, the old ADR should have it's status updated to point to the new ADR. The old ADR shouldn't be removed or otherwise modified, however, so we can have our historical record on why decisions were made.
+
+## 🎬 Consequences
+- Developers should write an ADR and submit it for review before selecting an approach to an architectural decision.
+- We'll have a concrete artifact for us to focus our discussion before finalizing decisions.
+- Decisions will be made deliberately, as a group.
+- We will have a useful set of documentation by default, providing a persisitent record of why decisions were made.
```
Comment (deathbearbrown) [inline docs/adr/0001-use-adrs.md:28]: Add a Changelog at the bottom for summarizing small modifications to the adr over time. ADRs are living docs.
it'd look like

### Changelog
-  mm/dd/yyyy proposed 
-  mm/dd/yyyy changed implementation from [..] to [..]

Comment (deathbearbrown) [reply docs/adr/0001-use-adrs.md:28]: Big changes would mean writing a new ADR

**Comment by deathbearbrown** [comment]
yaaaay

File: docs/adr/0001-use-adrs.md (hunk 1 above)
Comment (tkadlec) [reply docs/adr/0001-use-adrs.md]: Done! Thanks!

File: docs/adr/0001-use-adrs.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## 📝 Changelog
+- 06/15/2021 Proposed
+- 06/15/2021 Added Changelog
```
Comment (deathbearbrown) [inline docs/adr/0001-use-adrs.md:32]: LOL I see what you did there

Comment (tkadlec) [reply docs/adr/0001-use-adrs.md:32]: Trying to be thorough 😂

**Pull request merged**